dhcpv4/ztpv4/ztp.go

```diff
@@ -15,14 +16,39 @@ type VendorData struct {
 
 var errVendorOptionMalformed = errors.New("malformed vendor option")
 
+const entIDCiscoSystems = 0x9
+
 // ParseVendorData will try to parse dhcp4 options looking for more
 // specific vendor data (like model, serial number, etc).
 func ParseVendorData(packet *dhcpv4.DHCPv4) (*VendorData, error) {
+	vd := &VendorData{}
+	
+	vivc := packet.VIVC()
```

Comment:
This function is getting overloaded. Can you split things in different functions?

I think we would benefit from some code separation:
* A function for looking at the `VIVC` option
* A separate function for looking at the `ClassIdentifier` option
* And `ParseVendorData` calling both of them